marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.tsx

```diff
@@ -544,12 +573,11 @@ const SourceToEntityMap = (props) => {
         let mesg = `All changes are saved on ${convertDateFromISO(new Date())}`
         let errorMesg = `An error occured while saving the changes.`
 
-        let msg = <span id="successMessage"><Alert type="success" message={mesg} banner style={saveMessageCSS} /></span>
-        let errorMsg = <span id="errorMessage"><Alert type="error" message={errorMesg} banner style={saveMessageCSS} /></span>
+        let msg = <span data-testid="successMessage" id="successMessage"><Alert type="success" message={mesg} banner style={saveMessageCSS} /></span>
```

Comment:
Do we need a testid for successMessage. Will the test work if we expect(getByText('All changes are saved')). Lets try to use data-testid as last resort if nothing works.

Reply:
I did try it but getByText('All changes are saved') doesn't work for some reason even though debug prints this text. I am not sure why this happens with antd Alert messages , until there is another solution, I will go with this